### PR TITLE
Docs: added CSV profile references

### PR DESCRIPTION
**Pull request by mperezfuster**
Added references to CSV profile since this had not been documented:

https://mireia-pxf-csv-profile.sc2-04-pcf1-apps.oc.vmware.com/pxf/6-0/using/access_hdfs.html
https://mireia-pxf-csv-profile.sc2-04-pcf1-apps.oc.vmware.com/pxf/6-0/using/hdfs_text.html
https://mireia-pxf-csv-profile.sc2-04-pcf1-apps.oc.vmware.com/pxf/6-0/using/access_objstore.html
https://mireia-pxf-csv-profile.sc2-04-pcf1-apps.oc.vmware.com/pxf/6-0/using/access_s3.html

### CODE REVIEW

**Comment by mperezfuster** [comment]
@denalex I have made some changes to the docs, so now it only refers to hdfs:text profile for accessing/writing text, and hdfs:cvs for csv. The external table syntax and the table for parameter breakdown states that the parameter FORMAT needs to be specified, so I think there shouldn't be any confusion on it? Otherwise please let me know and I can change whatever you think it might mislead the user.

File: docs/content/access_hdfs.html.md.erb [hunk 1]
```diff
@@ -130,7 +132,7 @@ Choose the `hive` profile when:
 - The data resides in a Hive table, and you do not know the underlying file type of the table up front.
 - The data resides in a Hive table, and the Hive table is partitioned.
 
-Choose the `hdfs:text` profile when the file is text and you know the location of the file in the HDFS file system.
+Choose the `hdfs:text`, `hdfs:csv` profiles when the file is text and you know the location of the file in the HDFS file system.
```
Comment (dyozie) [inline docs/content/access_hdfs.html.md.erb:135]: , `hdfs:csv` profiles -> or `hdfs:csv` profile

File: docs/content/hdfs_text.html.md.erb [hunk 2]
```diff
@@ -29,12 +29,12 @@ Ensure that you have met the PXF Hadoop [Prerequisites](access_hdfs.html#hadoop_
 
 ## <a id="profile_text"></a>Reading Text Data
 
-Use the `hdfs:text` profile when you read plain text delimited or .csv data where each row is a single record. The following syntax creates a Greenplum Database readable external table that references such a text file on HDFS: 
+Use the `hdfs:text` profile when you read plain text delimited, and `hdfs:csv` when reading .csv data where each row is a single record. The following syntax creates a Greenplum Database readable external table that references such a text file on HDFS:
```
Comment (dyozie) [inline docs/content/hdfs_text.html.md.erb:32]: , and -> , or

**Review by dyozie — APPROVED** [review]

File: docs/content/hdfs_text.html.md.erb [hunk 3]
```diff
@@ -115,11 +115,11 @@ Beijing,Jul,411,11600.67' > /tmp/pxf_hdfs_simple.txt
     (4 rows)
     ```
 
-2. Create a second external table that references `pxf_hdfs_simple.txt`, this time specifying the `CSV` `FORMAT`:
+2. Create a second external table that references `pxf_hdfs_simple.txt`, this time specifying the `hdfs:csv` `PROFILE`:
```
Comment (denalex) [inline docs/content/hdfs_text.html.md.erb:118]:
```suggestion
2. Create a second external table that references `pxf_hdfs_simple.txt`, this time specifying the `hdfs:csv` `PROFILE` and the `CSV` `FORMAT`:
```

File: docs/content/objstore_text.html.md.erb [hunk 4]
```diff
@@ -139,11 +139,11 @@ Beijing,Jul,411,11600.67' > /tmp/pxf_s3_simple.txt
     (4 rows)
     ```
 
-2. Create a second external table that references `pxf_s3_simple.txt`, this time specifying the `CSV` `FORMAT`:
+2. Create a second external table that references `pxf_s3_simple.txt`, this time specifying the `s3:csv` `PROFILE`:
```
Comment (denalex) [inline docs/content/objstore_text.html.md.erb:142]: same here

**Pull request merged**